src/main/java/com/requestrealpiano/songrequest/service/SongService.java

```diff
@@ -18,16 +19,16 @@
     private final SearchApiService searchApiService;
 
     @Transactional
-    public Song findSongByRequest(SongRequest songRequest) {
-        return songRepository.findBySongTitleContainingIgnoreCaseAndArtistIgnoreCase(songRequest.getTitle(), songRequest.getArtist())
+    public Song updateRequestCountOrElseCreate(SongRequest songRequest) {
+        return songRepository.findBySongTitleAndArtist(songRequest.getTitle(), songRequest.getArtist())
```

Comment:
서비스 계층에서 보니 감이 옵니다. 만약 find메서드가 title이나 artist가 아니고, 년도나 앨범명이 추가될 수도 있는데 그때마다 메서드명을 바꾸게 된다면 어떻게 될까요?
그러면 해당 메서드명으로 된 것이 전부 다 바뀌겠죠? 

```suggestion
        return songRepository.find(songRequest));
```
로 처리한다면 중간에 넘기는 값이 바뀌더라도 find()메서드에서 쿼리만 바꿔주면 되니 변경도 적고 훨씬 유지보수하는데 큰 도움이 되겠죠?